src/routes/routes.js

```diff
@@ -148,12 +149,12 @@ const Routes = () => {
             )}
           />
           <Route path={routes.pathToContacts} exact component={ContactsPage} />
+          <Route path={routes.pathToAddContact} exact component={ContactsAdd} />
           <Route
             path={routes.pathToContactsEdit}
             exact
-            component={ContactsEdit}
+            render={({ match }) => <ContactsEdit id={match.params.id} />}
           />
```

Comment:
It will cause error in `ContactsEdit` component if it use `match`.

Reply:
fixed